### DISCUSSION TITLE
Welcome to maths-tripos-questions Discussions!

### DISCUSSION THREAD

**Discussion by AustenLamacraft** [tripos-education/maths-tripos-questions#1] [Announcements]
## 👋 Welcome!

This is the place to discuss where you'd like to see this site go, how you use it, and what features you'd like to see. 

If you've got a specific issue that needs addressing (something broken, say) consider [opening an issue](https://github.com/tripos-education/maths-tripos-questions/issues)

**Comment by callumhobbis** [comment]
Hi! I've been using this site for a bit, and I'd really like to see this site have a collaborative focus. It would be really useful if there were some method of submitting / voting on tags to give questions, and maybe some way to mark whether a question has a partial or full solution given by a commenter.

It motivates me to see things visually, so it might be nice to have a grid of squares representing each question (year vs question number) for a course, and then coloured based on whether a solution has been submitted (or maybe the number of comments), so you can see the progress towards covering every past question in a course.

Finally, a very niche thing that would be nice is a way to copy the Markdown of the question so I can copy it to something like Obsidian or Notion and preserve the LaTeX code! Thank you for the work you've put into making this site - it's really promising and already a valuable resource.

**Reply by AustenLamacraft** [reply]
Thanks for the ideas!

> Hi! I've been using this site for a bit, and I'd really like to see this site have a collaborative focus. It would be really useful if there were some method of submitting / voting on tags to give questions, and maybe some way to mark whether a question has a partial or full solution given by a commenter.

This stuff is available in GH Discussions, and so the question is what's available in the Discussions API. My impression was almost all the available functionality is used at the moment, but I'll look into it.

> It motivates me to see things visually, so it might be nice to have a grid of squares representing each question (year vs question number) for a course, and then coloured based on whether a solution has been submitted (or maybe the number of comments), so you can see the progress towards covering every past question in a course.

Sounds nice! There's definitely room for nicer visualisation.

> Finally, a very niche thing that would be nice is a way to copy the Markdown of the question so I can copy it to something like Obsidian or Notion and preserve the LaTeX code! 

I believe you should be able to do this already!

> Thank you for the work you've put into making this site - it's really promising and already a valuable resource.

No problem! Glad you're enjoying it!
